OpenRA.Mods.Common/Traits/GivesCashOnCapture.cs

```diff
@@ -0,0 +1,78 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Lets the actor grant cash when captured.")]
+	class GivesCashOnCaptureInfo : UpgradableTraitInfo
+	{
+		[Desc("Whether to show the cash tick indicators rising from the actor.")]
+		public readonly bool ShowTicks = true;
+
+		[Desc("How long to show the cash tick indicator when enabled.")]
+		public readonly int DisplayDuration = 30;
+
+		[Desc("Amount of money awarded for capturing the actor.")]
+		public readonly int CaptureAmount = 0;
```

Comment:
The `Capture` isn't needed anymore, so please rename this to `Cash` (to make it clear that it is not giving resources)